index.html

```diff
@@ -10,17 +10,29 @@
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+  <link href="https://allfont.net/allfont.css?fonts=crete-round" rel="stylesheet" type="text/css" />
 </head>
 <body>
```

Comment:
[ OPTIONAL ] Please create a directory named images and put all the image resources in it. It looks managed and nice when different directories are utilized.